implement-directus-auth-in-nextjs-14/index.md

```diff
@@ -0,0 +1,1105 @@
+---
+title: 'Implement Directus Auth in Next.js 14'
+description: 'A tutorial on implementing Authentication on a Next.js blog website using Directus as a Headless CMS.'
+author:
+  name: 'Emmanuel John'
+  avatar_file_name: 'profile-photo.jpeg'
+---
+
+## Introduction
+Next.js is a React.js server-side rendering framework that allows front-end developers to build full-stack applications with a single codebase. It has turborpack the rust-based bundler built-in for enhanced code optimization and fast code compilation. It’s a great choice for performance, efficiency and developer experience. In this tutorial, you will learn how to implement Authentication on a Next.js blog website using Directus as a Headless CMS. You’ll register new users, login, handle refresh tokens, create posts, list all posts, and edit/delete posts where the auth user is the post creator.
+
+## Before You Start
+To follow along with this tutorial, you will need the following knowledge and tools:
+
+- [Node.js](https://nodejs.org/en/download/) v18 and above installed on your computer.
+- A Directus project - you can use [Directus Cloud or run it yourself](https://docs.directus.io/getting-started/quickstart).
+- Prior experience with Nextjs.
```

Comment:
```suggestion
- Prior experience with Next.js.
```

Can you go through the whole article and make sure you're properly stylizing "Next.js" please?